core/db/models.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+
+from sqlalchemy import func, String, ForeignKey
```

Comment:
Let's sort imports to make it easier to read. It isnt very important but it's better to do it

Reply:
I'll sort them now, but it'll get messy again when this file will be changed. Perhaps make a rule that before every commit it automatically checks stuff? I think it's called pre-commit hook